indico/modules/events/registration/client/js/form/fields/TextInput.jsx

```diff
@@ -27,6 +27,7 @@ export default function TextInput({htmlName, disabled, title, isRequired, ...pro
       {...inputProps}
       required={isRequired}
       disabled={disabled}
+      parse={p.nullIfEmpty}
```

Comment:
We have a prop for this in FinalInput:

```suggestion
      nullIfEmpty
```

(and no need to import `p` then)

What's the reason for sending an empty text field as `null` btw? Currently we store an empty string and never `null` if the field is empty...

Reply:
exactly. is there a cleaner way to go around that? otherwise fields with a min length set are always required

Reply:
I don't think so... I don't like the idea that we are suddenly switching from storing an empty string to storing null.

One option could be subclassing the `Length` validator and overriding its `__call__` method to return early `if not value`

Reply:
Actually, I think a minimum length of 1 makes no sense at all. If people want to require input, the field should be required. If people do not want single-char values, they can set a higher maxlength